src/domain/app/apollo/apolloClient.ts

```diff
@@ -1,31 +1,71 @@
 import { ApolloClient, ApolloLink, InMemoryCache } from '@apollo/client';
 import { setContext } from '@apollo/client/link/context';
 import { RestLink } from 'apollo-link-rest';
+import snakeCase from 'lodash/snakeCase';
 
 import {
   Event,
   EventsResponse,
+  Image,
+  ImagesResponse,
   Keyword,
   KeywordSet,
   KeywordSetsResponse,
   KeywordsResponse,
   LanguagesResponse,
   Place,
   PlacesResponse,
+  UploadImageMutationInput,
 } from '../../../generated/graphql';
 import { normalizeKey } from '../../../utils/apolloUtils';
 import { apiTokenSelector } from '../../auth/selectors';
 import i18n from '../i18n/i18nInit';
 import { store } from '../store/store';
 import {
   addTypenameEvent,
+  addTypenameImage,
   addTypenameKeyword,
   addTypenameKeywordSet,
   addTypenameLanguage,
   addTypenameMeta,
   addTypenamePlace,
 } from './utils';
 
+const uploadImageSerializer = (
```

Comment:
Maybe this could use small description why this is needed? what do you think?

Reply:
Added a short description